include/boost/math/tools/config.hpp

```diff
@@ -84,6 +84,8 @@
 
 #else // Things from boost/config that are required, and easy to replicate
 
+#include <version>
```

Comment:
```suggestion
#if __has_include(<version>)
#include <version>
#endif
```

This wasn't standardized until C++20. I think it's unofficially available earlier, but looks like at least some of the older clangs don't have it.